src/TabCompleteEntries.js

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2015 OpenMarket Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+var React = require("react");
```

Comment:
Yup. `sdk` is used on `:60`. `React` needs to be defined because the JSX compiler is *hilarious* and will bluntly replace in-line HTML with `React.createClass` and if you don't have `React` as a variable... **kaboom!** We use JSX on `:62`.